HW02_B_Mimic_starter.Rmd

```diff
@@ -42,6 +40,10 @@ data("diamonds")
 Using the diamonds dataset, make this graph:
 ```{r graph1 code, echo=FALSE}

```

Comment:
I would recommend changing "echo" to TRUE, so that codes will appear right above the graphs in the corresponding  .md file, which makes it easier for others to review them!

Reply:
Thank you! I couldn't really remember what that function did, so I just left it as I saw it in anothe example.